src/util/input_method_editor.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+#include "common.hpp"
+#include "pointers.hpp"
+
+#include <imgui_internal.h>
+
+#pragma pack(push, 8)
+class InputMethodEditor
+{
+public:
+	uint32_t m_count; //0x0000
+	uint32_t m_selected_index; //0x0004
+	wchar_t m_composition_string[31]; //0x0008
+	wchar_t m_candidate_list[9][31]; //0x0046
+	bool m_active; //0x0274
+	char pad_0275[3]; //0x0275
+}; //Size: 0x0278
+static_assert(sizeof(InputMethodEditor) == 0x278);
+#pragma pack(pop)

```

Comment:
Since this class is coming from GTA 5 itself, it belongs in the GTAV-Classes project.